src/pokemon_pb.erl

```diff
@@ -87,7 +87,7 @@ pack(FNum, repeated_packed, Data, Type, _) ->
     protobuffs:encode_packed(FNum, Data, Type);
 
 pack(FNum, _, Data, _, _) when is_tuple(Data) ->
-    [RecName|_] = tuple_to_list(Data),
+    RecName = erlang:element(1, Data),
```

Comment:
This seems like a good change.

@javajolt @alexmoore @tburghart - care to weigh in?